### PR TITLE
avatar created

### PR DESCRIPTION
**Pull request by arundhatij12**

### CODE REVIEW

File: avatar.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
+body{
+    background-color: rgb(235, 227, 227);
```
Comment (Karan-Bharti) [inline avatar.css]: Try to put colors and font size in style css file .i.e. you would be able to use these colors and fonts again as your stylesheet would be linked with your all the components

File: avatar.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <title>Avatar</title>
+    <link rel="stylesheet" href="avatar.css">
+</head>
+<body>
+
+    <header>
+        <div class="head">
+            <div><a class="title" href=""> <h2>component library</h2></a>
+            </div>
+            <nav class="">
```
Comment (Karan-Bharti) [inline avatar.html]: Why this empty nav is used?

**Review by Karan-Bharti — COMMENTED** [review]
Try to look on my points i mentioned

**Review by kjais1720 — CHANGES_REQUESTED** [review]
Try not to use hard coded values for colors, instead create variables for all the colors and use those variables.
Also, create utility classes for styles that are being repeated at many places (like the `display: flex;`)

**Pull request merged**